tests/integration/targets/iosxr_banner/tasks/main.yaml

```diff
@@ -1,8 +1,3 @@
 ---
 - include: cli.yaml
-  tags:
-    - cli
```

Comment:
this should be network_cli

Reply:
This is already in place. 'cli' tag is removed.